R/configuration-plan.R

```diff
@@ -231,6 +240,9 @@ ConfigurationPlan <- R6::R6Class(
         path = sapply(value, function(mapping) {
           mapping$Path
         }),
+        molWeight = sapply(value, function(mapping) {
+          mapping$MolWeight %||% mapping$MW %||% NA

```

Comment:
I don't know yet how the Qualification Runner will name such a field, but if necessary I can update here.
Currently, this reads as look for a field named "MolWeight", then "MW" if nothing molWeight is NA